rubin_sim/maf/generate_ss.py

```diff
@@ -82,9 +94,12 @@ def generate_ss_commands(
             for run, filename in zip(runs, dbfiles):
                 objtype = objtypes[pop]
 
-                s1 = f"make_lsst_obs --simulation_db {filename} --orbit_file {orbit_files[pop]}"
+                s1 = (
+                    f"make_lsst_obs --simulation_db {filename} --orbit_file {orbit_files[pop]}"
+                    f" --positions_file {position_files[pop]}"
+                )
```

Comment:
These are relatively minor changes, can you please add them to the "split" processing option as well? 
(I am curious how split processing might work if we have lots of cores and can more interactively split the orbit files into multiple subsets depending on how many workers are available -- i.e. not just 10, but maybe 100 subsets -- and reduce the processing time per simulation. In the future we're likely to have many fewer simulations, and need to turn them around faster).

Reply:
Sure. I think we can simplify the scatter/gather process for running things in parallel, but that can be a different ticket.